packages/wireguard-tools/build.sh

```diff
@@ -5,11 +5,4 @@ TERMUX_PKG_SRCURL=https://git.zx2c4.com/WireGuard/snapshot/WireGuard-$TERMUX_PKG
 TERMUX_PKG_SHA256=096b6482a65e566c7bf8c059f5ee6aadb2de565b04b6d810c685f1c377540325
 TERMUX_PKG_DEPENDS="libmnl"
 TERMUX_PKG_BUILD_IN_SRC=yes
-
-termux_step_make() {
-  make -C src/tools WITH_BASHCOMPLETION=yes WITH_WGQUICK=no WITH_SYSTEMDUNITS=no
-}
-
-termux_step_make_install() {
-  make -C src/tools install PREFIX=$TERMUX_PREFIX WITH_BASHCOMPLETION=yes WITH_WGQUICK=no WITH_SYSTEMDUNITS=no
-}
+TERMUX_PKG_EXTRA_MAKE_ARGS=" -C src/tools WITH_BASHCOMPLETION=yes WITH_WGQUICK=no WITH_SYSTEMDUNITS=no"
```

Comment:
Is the space after " necessary?

Reply:
in case other case get specified elsewhere, the space if good to make sure that concatenation goes smoothly